source/NVDAObjects/UIA/__init__.py

```diff
@@ -2265,15 +2290,9 @@ def event_stateChange(self):
 
 class SearchField(EditableTextWithSuggestions, UIA):
 	"""An edit field that presents suggestions based on a search term.
+	This is now an empty class as Functionality has been moved to the base EditableText behaviour.

```

Comment:
Fix mis-capitalization in functionality

```suggestion
	This is now an empty class as functionality has been moved to the base EditableText behaviour.
```